.env.example

```diff
@@ -32,8 +32,8 @@ AWS_S3_REORDER_POINTS_MULTIPLIERS_BUCKET=
 AWS_SQS_ACCESS_KEY_ID=stockup
```

Comment:
Add minio default clientId secret and bucket names in this file

Reply:
Done !